public/app/features/dashboard/components/DashboardSettings/VersionsSettings.tsx

```diff
@@ -6,12 +6,13 @@ import { Page } from 'app/core/components/Page/Page';
 import {
   historySrv,
   RevisionsModel,
-  VersionHistoryTable,
   VersionHistoryHeader,
   VersionsHistoryButtons,
-  VersionHistoryComparison,
 } from 'app/features/dashboard-scene/settings/version-history';
 
+import { VersionHistoryComparison } from '../VersionHistory/VersionHistoryComparison';
+import { VersionHistoryTable } from '../VersionHistory/VersionHistoryTable';
```

Comment:
Since we're changing the restore logic in the the restore modal I've decided to re-add these 2 files to the old `VersionHistory` folder and have two copies, one which uses the old restore modal and the `scenes/version-history` ones which use a refactored restore modal adapted to scenes.